package.json

```diff
@@ -144,11 +144,11 @@
     },
     "peerDependencies": {
         "@emotion/react": "^11.1.5",
-        "@guardian/commercial-core": "^4.25.0",
-        "@guardian/consent-management-platform": "^10.5.0",
-        "@guardian/libs": "^9.0.1",
-        "@guardian/source-foundations": "^7.0.1",
-        "@guardian/source-react-components": "^9.0.1",
+        "@guardian/commercial-core": "^5.0.0",
+        "@guardian/consent-management-platform": "^11.0.0",
+        "@guardian/libs": "^10.1.1",
```

Comment:
does this rely on the ophan type added in 10.1? https://github.com/guardian/csnx/blob/main/libs/%40guardian/libs/CHANGELOG.md#1010